src/primitives/felt.ts

```diff
@@ -19,14 +19,14 @@ export class Felt {
   }
 
   add(other: MaybeRelocatable): Felt {
-    if (!(other instanceof Felt)) {
-      throw new PrimitiveError(ForbiddenOperation);
+    if (MaybeRelocatable.isFelt(other)) {
+      return new Felt((this.inner + other.inner) % Felt.PRIME);
     }
-    return new Felt((this.inner + other.inner) % Felt.PRIME);
+    throw new PrimitiveError(ForbiddenOperation);
```

Comment:
Throw error in the `if`condition, as in `sub()` and `div()` for consistency and readability

Reply:
Fixed, it was to avoid a `!` statement but sounds reasonable, just fixed.